dtbs/bcm2837-rpi-3-b-plus.dts

```diff
@@ -17,13 +18,20 @@
 			vmid = <0>;
 			vm_name = "rpi3_linux_host";
 			type = "linux";
-			vcpus = <2>;
```

Comment:
here should not changed this line.

Reply:
Sorry, I increased the number of cores to improve the performance of the guest Linux. I will revert it.